examples/website/i3s/components/toggle-switch.js

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import styled from "styled-components";
+
+const Input = styled.input`
+  height: 0;
+  width: 0;
+  opacity: 0;
+  z-index: -1;
+`;
+
+const Label = styled.label`
+  position: absolute;
+  display: inline-block;
```

Comment:
`display: inline-block` and `position: absolute` simultaneously doesn't have sense.